src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/SortBamSparkIntegrationTest.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.hellbender.tools.spark.pipelines;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.read.SamAssertionUtils;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public final class SortBamSparkIntegrationTest extends CommandLineProgramTest {
+
+    @Test
+    public void test() throws Exception {
+        final File unsortedBam = new File(getTestDataDir(), "count_reads.bam");
+        final File sortedBam = new File(getTestDataDir(), "count_reads_sorted.bam");
+        final File outputBam = createTempFile("sort_bam_spark", ".bam");
+        outputBam.delete();
```

Comment:
ReadsSparkSink doesn't overwrite existing files, so it will fail since `createTempFile` actually creates the output file. Should we change the sink to overwrite?